config/routes.rb

```diff
@@ -1,11 +1,13 @@
 Rails.application.routes.draw do
 
+
   devise_for :users
   resources :users, only: [:update]
    
-   resources :topics do
-     resources :posts, except: [:index]
-   end
+   resources :topics, :posts, :comments do
+   resources :posts, except: [:index]
+   resources :comments, only: [:create]
```

Comment:
Ok so posts is nested under topics, but comments needs to be nested under posts as well.